taxonomies/ingredients.txt

```diff
@@ -69076,6 +69087,11 @@ allergens:en: en:eggs
 carbon_footprint_fr_foodges_ingredient:fr:Oeuf, moyenne nationale
 carbon_footprint_fr_foodges_value:fr:2.6
 
+<en:egg
+en:pasteurized egg, pasteurized eggs
+da:pasteuriseret æg
+de:pasteurisiertes Ei, pasteurisierte Eier
+
```

Comment:
```suggestion
```
This is why the test fails. Pasteurized is in processing. It's not yet enabled for Danish (I'm working on finishing the tests for it)

Reply:
Okay great. I am often unsure if a specific combination of ingredient and processing should get their own entry in the ingredients.txt. Because there are a lot of these combinations but they are often not really necessary as far as I know.

Reply:
Historically, we were not able to parse the processing, so we added entries like "pasteurized eggs". But now that we parse the processing, we should remove those combined entries. (the tricky part is to make sure we have working processing parsing for as many languages as possible)

Reply:
Strange couldn't commit through the browser. So I should definitely not add new one and I could delete old ones but it has no high priority since the don't cause any harm, right? @stephanegigandet

Reply:
That's right. We need to be careful when we delete entries though, as they may be covered by the processing algorithm/taxonomies for some languages, but not all the ones for the combined entry. If possible we should then add processing (but then we have also to be very careful about false positives) for more languages.